srcjs/output_binding_image.js

```diff
@@ -96,74 +93,91 @@ $.extend(imageOutputBinding, {
     if (!opts.coordmap)
       opts.coordmap = [];
 
-    imageutils.initCoordmap($el, opts.coordmap);
+    // When the image loads, initialize all the interaction handlers. When the
+    // value of src is set, the browser may not load the image immediately,
+    // even if it's a data URL. If we try to initialize this stuff
+    // immediately, it can cause problems because we use .naturalWidth and
+    // .naturalHeight, but they're available only after the image loads.
+    $img.off("load.shiny-image-interaction");
+    $img.on("load.shiny-image-interaction", function() {
```

Comment:
Investigate getting rid of this delay by having the server return the image width/height